conda/activate.py

```diff
@@ -194,12 +194,10 @@ def hook(self, auto_activate_base: bool | None = None) -> str:
             builder.append(preamble)
         if self.hook_source_path:
             builder.append(self.hook_source_path.read_text())
-        if (
-            auto_activate_base is None
-            and context.auto_activate_base
-            or auto_activate_base
-        ):
-            builder.append("conda activate base\n")
+        if auto_activate_base is None and context.auto_activate or auto_activate_base:
```

Comment:
```suggestion
        if auto_activate is None and context.auto_activate or auto_activate:
```
And either rename the method parameter or create a new variable in the block with the preferred name

Reply:
Ok went for it and deprecated the old one.